casper/src/test/scala/coop/rchain/casper/api/PreviewPrivateNameTest.scala

```diff
@@ -1,55 +0,0 @@
-package coop.rchain.casper.api
```

Comment:
We will remove private names from API completely so this test becomes irrelevant.